src/RockPaperScissors.cpp

```diff
@@ -0,0 +1,43 @@
+#include "RockPaperScissors.h"
+#include <string>
+using namespace std;
+
+string RockPaperScissorsPlay(const string player1, const string player2)
+{
+    if (player1 == "Scissors" && player2 == "Rock")
+    {
+        return "Player1 Loses";
+    }
+    if (player1 == "Scissors" && player2 == "Paper")
+    {
+        return "Player1 wins";
+    }
+    if (player1 == "Scissors" && player2 == "Scissors")
```

Comment:
There's a lot of `if (player1 == blah && player2 == blah)` in this file. Can you think of any alternate ways to structure this code that would mean less copy-pasting?